File: dpo_train/generation_output/py-java/HumanEval_141_file_name_check.java
### Python version

def file_name_check(file_name: str) -> str:
    suf = ['txt', 'exe', 'dll']
    lst = file_name.split(sep='.')
    if len(lst) != 2:
        return 'No'
    if not lst[1] in suf:
        return 'No'
    if len(lst[0]) == 0:
        return 'No'
    if not lst[0][0].isalpha():
        return 'No'
    t = len([x for x in lst[0] if x.isdigit()])
    if t > 3:
        return 'No'
    return 'Yes'

### Java version

import java.util.*;
import java.lang.reflect.*;
import org.javatuples.*;
import java.security.*;
import java.math.*;
import java.io.*;
import java.util.stream.*;
class Problem {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem2 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem3 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem4 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem5 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem6 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem7 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem8 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem9 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem10 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem11 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem12 {
    public static String fileNameCheck(String file_name) {
        if (file_name.endsWith(".java")) {
            return file_name;
        } else {
            return file_name + ".java";
        }
    }
}

class Problem

File: dpo_train/generation_output/py-cpp/HumanEval_141_file_name_check.cpp
### Python version

def file_name_check(file_name: str) -> str:
    suf = ['txt', 'exe', 'dll']
    lst = file_name.split(sep='.')
    if len(lst) != 2:
        return 'No'
    if not lst[1] in suf:
        return 'No'
    if len(lst[0]) == 0:
        return 'No'
    if not lst[0][0].isalpha():
        return 'No'
    t = len([x for x in lst[0] if x.isdigit()])
    if t > 3:
        return 'No'
    return 'Yes'

### C++ version

#include<assert.h>
#include<bits/stdc++.h>
#include<vector>
#include<sstream>
#include<regex>
#include<map>
#include<set>
#include<unordered_map>
#include<unordered_set>
std::string file_name_check(std::string file_name) {
    std::stringstream ss;
    ss << file_name;
    std::string file_name_str = ss.str();
    std::string file_name_str_2 = file_name_str.substr(0, file_name_str.find('.'));
    std::string file_name_str_3 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_4 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_5 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_6 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_7 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_8 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_9 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_10 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_11 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_12 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_13 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_14 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_15 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_16 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_17 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_18 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_19 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_20 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_21 = file_name_str.substr(file_name_str.find('.') + 1);
    std::string file_name_str_22 = file_name_str.substr(file_name_str.find('.'